solutions/swb-reference/integration-tests/tests/multiStep/et-etc.test.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+import { resourceTypeToKey } from '@aws/workbench-core-base';
+import ClientSession from '../../support/clientSession';
+import { EnvironmentTypeHelper } from '../../support/complex/environmentTypeHelper';
+import Setup from '../../support/setup';
+import { envTypeConfigRegExp } from '../../support/utils/regExpressions';
+
+describe('multiStep environment type and environment type config test', () => {
+  const setup: Setup = new Setup();
+  let adminSession: ClientSession;
+  const envTypeHandler = new EnvironmentTypeHelper();
+  beforeAll(async () => {
+    adminSession = await setup.getDefaultAdminSession();
+  });
+
+  afterAll(async () => {
+    await setup.cleanup();
+  });
+
+  test('create Environment Type', async () => {
+    //Create Environment A
+    console.log('Creating Environment Type');
+    const envType = await envTypeHandler.createEnvironmentType('prod-1234567890123', 'pa-1234567890123');
+    const expectedId = `${resourceTypeToKey.envType.toLowerCase()}-prod-1234567890123,pa-1234567890123`;
+    expect(envType).toMatchObject({
+      id: expectedId,
+      status: 'NOT_APPROVED'
+    });
+
+    //Approve Environment Type
+    console.log('Approve Environment Type');
+    await adminSession.resources.environmentTypes.environmentType(envType.id).update(
```

Comment:
Are there any other fields to test updating?

Reply:
yes , description and name, I am planning to add 2 more tests updating only one property at a time when I fix the issue that requires status on ET patch

Reply:
added description and name PATCH to integration test